processor/tailsamplingprocessor/sampling/string_tag_filter.go

```diff
@@ -90,3 +97,31 @@ func (saf *stringAttributeFilter) Evaluate(_ pdata.TraceID, trace *TraceData) (D
 	}
 	return NotSampled, nil
 }
+
+// addFilters compiles all the given filters and stores them as regexes.
+// All regexes are automatically anchored to enforce full string matches.
+func addFilters(exprs []string) []*regexp.Regexp {
+	list := make([]*regexp.Regexp, 0, len(exprs))
+	for _, entry := range exprs {
+		rule := regexp.MustCompile(entry)
```

Comment:
Are we certain that all the exprs are valid regex?  If not we can call `Compile` and ignore the error (assuming there is no way to propagate this error back to caller)

Reply:
It is called in `createTracesProcessor` when the processor is initialized. `MustCompile` will `panic` if the regex in configuration is not valid. I think the next step is to add `config.Validate()`.

Reply:
Do you plan to add the validate logic in this PR?